docs/deployment/downgrade.md

```diff
@@ -35,15 +35,35 @@ During preparation, you must perform the compatibility configuration if you are
 
 ### Perform compatibility configuration
 
-If you want to downgrade your StarRocks cluster to an earlier minor or major version, you must perform the compatibility configuration. Compatibility configuration varies from the version of the StarRocks cluster you downgrade from.
+If you want to downgrade your StarRocks cluster to an earlier minor or major version, you must perform the compatibility configuration. In addition to the universal compatibility configuration, detailed configurations vary depending on the version of the StarRocks cluster you downgrade from.
 
-#### From v2.2 and later versions
+- **Universal compatibility configuration**
+
+Before downgrading your StarRocks cluster, you must disable tablet clone.
+
+```SQL
+ADMIN SET FRONTEND CONFIG ("max_scheduling_tablets" = "0");
+ADMIN SET FRONTEND CONFIG ("max_balancing_tablets" = "0");
+ADMIN SET FRONTEND CONFIG ("disable_balance"="true");
+ADMIN SET FRONTEND CONFIG ("disable_colocate_balance"="true");
+```
+
+After the downgrade, and the status of all BE nodes is `Alive`, you can re-enable tablet clone.
```

Comment:
After the downgrade, you can enable tablet clone again if the status of all BE nodes becomes `Alive`.